event-based-simulators/main/cumulocityAPI.py

```diff
@@ -48,14 +69,16 @@ def get_or_create_device(self, sim_id, label):
         if self.mocking:
             log.info(f"mock: get or create device with external id {sim_id}")
             return sim_id
-        
+
```

Comment:
see first comment rgd irrelevant changes; also applies to a few more changes below